src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptorProvider.cs

```diff
@@ -14,6 +14,8 @@ public class TagHelperDescriptorProvider
     {
         public const string CatchAllDescriptorTarget = "*";
 
+        public static readonly string RequiredAttributeWildcardSuffix = "*";
```

Comment:
The `CatchAllDescriptorTarget` is purposefully `public const` so that the `TargetElement` can expose it as a `public const` for users to consume. The `RequiredAttributeWildcardSuffix` isn't supposed to be easily consumable by users so it's just here as `public static readonly` so we can use the same value throughout our code.